docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -23,6 +23,18 @@ Since Yugabyte Platform and the LDAP server are synchronized during login, Yugab
 
 If configured by the LDAP server, Yugabyte Platform can prevent the user from being able to change their password.
 
+## Use the Yugabyte Platform UI Console
+
+You can use the Yugabyte Platform UI console to enable LDAP authentication for Yugabyte Platform login, as follows:
+
+- Navigate to **Admin > User Management > User Authentication**. 
+
+- Complete the fields in the **LDAP Configuration** page shown in the following illustration:<br><br>
+
+  ![LDAP authentication](/images/yp/ldap-auth-1.png)<br><br>
+
+  The description of these settings are provided in [Use the Yugabyte Platform API](#use-the-yugabyte-platform-ui).
```

Comment:
We cannot use API description for "LDAP URL" field because we don't have a separate field for port number in the UI.


LDAP URL field will be combination of both "LDAP_URL" & "LDAP_PORT" ( colon separated )
Ex: 0.0.0.0:9000 ( <LDAP_URL>:<LDAP_PORT> )

Reply:
Fixed